docs/topics/js/js-project-setup.md

```diff
@@ -240,7 +240,7 @@ Once an npm dependency is installed, you can use its API in your code as describ
 
 ## run task
 
-The Kotlin/JS plugin provides a `jsRun` task that lets you run pure Kotlin/JS projects without additional configuration.
+The Kotlin Multiplatform Gradle plugin provides a `jsRun` task that lets you run pure Kotlin/JS projects without additional configuration.
```

Comment:
JFYI
looks like the plugin name is changed in all the other docs, but missed in this doc